documentation/design/software_design.md

```diff
@@ -0,0 +1,39 @@
+# **Gitops Plugin Design**
+Gitops software will provide ability to monitor git repositories for commits , changes etc and send the data to observability platforms like snappyflow.
```

Comment:
Use consistent naming. Repo says git audit, here we say Gitops software.

Reply:
done